### DISCUSSION TITLE
DeePMD-kit(2.0.2) to evaluate force

### DISCUSSION THREAD

**Discussion by zjd11** [deepmodeling/deepmd-kit#1186] [Q&A]
When i use the python interface of DeePMD-kit(2.0.2) to evaluate force,and the following error error has occurred.
File "/public/home/nsccsz_user75/bin/deepmd-kit-202/lib/python3.9/site-packages/numpy/core/fromnumeric.py", line 58, in _wrapfunc
    return bound(*args, **kwds)
ValueError: cannot reshape array of size 3072 into shape (432)

the python interface can be correctly used to evaluate force when the deepmd-kit version is 2.0.1. 
so ,how to evaluate force by using deepmd-kit-2.0.2?

**Comment by njzjz** [comment]
Could you provide your script and entire traceback?
I don't get errors when following [the example](https://docs.deepmodeling.org/projects/deepmd/en/v2.0.2/api_py/deepmd.infer.html#module-deepmd.infer.deep_pot).

**Reply by zjd11** [reply]
When i use deepmd-kit-2.0.1, i write a script as following to evaluate force, and it can obtain correct results. the obtained forces are almost the same as that obtain from deepmd-lammps, but when i use the script to deepmd-kit-2.0.2, it doesn't work. 

from deepmd.infer import DeepPot
import numpy as np
dp = DeepPot('nnpa.pb')
coord = np.loadtxt('./coord.raw')
cell = np.loadtxt('./box.raw')
atype = np.loadtxt('./type.raw').reshape(1,-1)
e, f, v = dp.eval(coord, cell, atype)
np.savetxt('f1.txt',f.reshape(-1,3))

**Reply by njzjz** [reply]
`atype` should be a 1-D list but not 2D array.
